_episodes/01-introduction.md

```diff
@@ -4,49 +4,106 @@ teaching: 5
+It is a platform for developing, shipping, and running applications in containers.
+In addition, Docker provides a public registry for sharing and
+collaborating on container images called [Docker Hub][docker-hub].
 
 The [official Docker documentation and tutorial][docker-tutorial] can be found on the
 Docker website.
-It is quite thorough and useful.
-It is an excellent guide that should be routinely visited, but the emphasis of this
-introduction is on using Docker, not how Docker itself works.
-
+It is quite thorough and useful, and an excellent guide that should be routinely visited when working with Docker.
 A note up front, Docker has very similar syntax to Git and Linux, so if you are familiar
 with the command line tools for them then most of Docker should seem somewhat natural
 (though you should still read the docs!).
 
-<img src="../fig/Docker_logo.png" width="200" alt="Docker logo" />
 
 It is still important to know what Docker _is_ and what the components of it _are_.
 Docker images are executables that bundle together all necessary components for an
 application or an environment.
 [Docker containers][docker-containers] are the runtime instances of images &mdash; they
 are images with a state.
 
-Importantly, containers share the host machine's OS system kernel and so don't require an
-OS per application.
-As discrete processes containers take up only as much memory as necessary, making them
-very lightweight and fast to spin up to run.
+Docker is the most popular containerization tool these days, but it's not the only one.
+There are other kids on the block which are in use and gaining popularity, such as Podman.
 
-<img src="../fig/DockerVM.png" alt="DockerVM" style="width:800px">
+# Why Podman?
 
-> ## Singularity
-> Docker is the most popular containerization tool these days, particularly in industry, but it's not the only one. There are other kids on the block including Rocket and Singularity which are in use, but just haven't gained as much large-scale traction.
+<img src="../fig/podman_logo.svg" width="400" alt="Docker logo" />
+
+Podman is an open-source alternative to Docker with several advantages.
+For example, Podman is able to run containers as a non-root user out of the box, which is a big security advantage over Docker.
+The reason is Podman uses a servless architecture, which means that it doesn't require a daemon running as superuser
+to execute containers as Docker does.
+
+In addition, Docker Desktop has [licensing restrictions](https://www.docker.com/pricing/) that may prevent you
+from using it in some institutions. If that is your case, Podman is an excellent alternative.
+
+Podman is a drop-in replacement for Docker, so you can use the same commands and workflows you are used to with Docker.
+Across the tutorial, we will use Podman as the containerization tool, but
+**if you are interested in using Docker instead, just replace `podman` by `docker` in the commands and you should be good to go**.
```

Comment:
This makes it sound like the following tutorials will show `podman` commands and that students could substitute `podman` -> `docker`. In fact, the tutorials _still_ use `docker` commands, so students will have just been told the tutorials use podman but work with docker when, in fact, it's the other way around.

Reply:
Sorry for the confusion. Yes, as you pointed out the commands beyond the introduction are addressed by #85